Lib/Rabbit4000/Crypto/MD4.LIB

```diff
@@ -75,6 +92,8 @@ __xmem void md4_round3(void);
 	   exp $ \

```

Comment:
Wouldn't it be better to use identical code on 4000/6000 and just make use of the `alts` prefix?

Reply:
The Rabbit 2000-5000 instruction guide does not mention alts or altsd so I avoided them in the non 6000 code. If alts is ok to use then I can clean things up a bit. Also I haven't checked it properly but "altsd ld pd, ps+n does not seem to produce the result I expected so I avoided it until I have a chance to experiment a bit further.
I've added a second work sheet to the r6K spreadsheet with the instructions ordered alphabetically and the new R6000 instructions highlighted which I find useful .
[Rabbit 6000 Instruction Spreadsheet.xlsx](https://github.com/digidotcom/DCRabbit_10/files/7370939/Rabbit.6000.Instruction.Spreadsheet.xlsx)